### PR TITLE
bump reva and change nats port

### PR DESCRIPTION
**Pull request by kobergj**
Signed-off-by: jkoberg <jkoberg@owncloud.com>

## Description
Change Nats port and bump reva version

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiArchiver-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/9528/21/1) failed. Further test are cancelled...

File: storage/pkg/command/sharing.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func sharingConfigFromStruct(c *cli.Context, cfg *config.Config) map[string]inte
 				"eventsmiddleware": map[string]interface{}{
 					"group":     "sharing",
 					"type":      "nats",
-					"address":   "127.0.0.1:4222",
+					"address":   "127.0.0.1:9233",
```
Comment (rhafer) [inline storage/pkg/command/sharing.go:182]: Shouldn't this be configurable as well?

**Review by rhafer — APPROVED** [review]
I am fine with merging this as it doesn't add a regression, but I guess we need to make the nats connection configurable in the storage service as well.

File: storage/pkg/command/sharing.go (hunk 1 above)
Comment (kobergj) [reply storage/pkg/command/sharing.go:182]: This will be done with this PR: https://github.com/owncloud/ocis/pull/3214

**Pull request merged**